test/json-ld.net.tests/DockerTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq.Expressions;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace JsonLD.Test
+{
+    public class DockerTests
+    {
+        private readonly ITestOutputHelper output;
+        
+        public DockerTests(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        [SkippableFact]
+        public void DockerBuild()
+        {
+            try
+            {
+                var exit = ExecDocker("build -t json-ld.net .");
```

Comment:
Wouldn't it be easier to just do `docker build -t json-ld.net .` in the `.github/workflows/dotnet.yml` file, or perhaps in a new `docker.yml` file? Then it would execute in parallel. The only downside is that it would be slightly harder to run locally, but as long as it is run on all pushes to GitHub, I don't think that's a huge downside.

Reply:
Fair enough. Done.